CONTRIBUTING.md

```diff
@@ -327,6 +327,9 @@ lines is greater than defined below.
 
 | Label | Description |
 | ----- | ----------- |
-| `size/small` | Anything less than or equal to 4 files and 150 lines. Only small amounts of manual testing may be required |
-| `size/medium` | Anything greater than `size/small` and less than or equal to 8 files and 300 lines. Manual validation should be required. |
-| `size/large` | Anything greater than `size/medium`. This should be thoroughly tested before merging and always requires 2 approvals. This also should be applied to anything that is a significant logic change. |
+| `size/XS` | Anything less than or equal to 9 lines ignoring generated files. Only small amounts of manual testing may be required |
+| `size/S` | Anything greater than `size/XS` less than or equal to 29 lines ignoring the generated files. Only small amounts of manual testing may be required |
+| `size/M` | Anything greater than `size/S` less than or equal to 99 lines ignoring the generated files. This should be thoroughly tested before merging and always requires 1 approvals. Manual validation should be required. |
```

Comment:
Can you remove "This should be thoroughly tested before merging and always requires 1 approvals"? I feel that the second line (manual validation should be required) covers the first statement, and every PR requires at least 1 approval so its's moot to call that out here.